base/linalg/generic.jl

```diff
@@ -417,48 +417,38 @@ istriu(x::Number) = true
 istril(x::Number) = true
 isdiag(x::Number) = true
 
-#TODO: lookup where all the documentation for linreg is coming from, including the example
-# mentioning PyPlot etc
 """
     linreg(x, y)
 
 Find the Ordinary Least Squares best linear fit between the numerical vectors `x` and `y`
 
 Inputs:
 
-- `x::AbstractVector{T <: Number}` - vector of independent variables
-- `y::AbstractVector{S <: Number}` - vector of dependent variables
+- `x::AbstractVector` - vector of independent variables
+- `y::AbstractVector` - vector of dependent variables
 
 Output:
 
-- `[b0, b1]` - the coefficients for the line of best fit such that yhat = b0 + b1*x
+- `(a, b)` - the coefficients for the line of best fit satisfying a + b*x
```

Comment:
I am happy to use either, see if there is any consensus. I changed this so that it was consistent with the example given in the `Base.rst` file. Changing to the parentheses is meant to be consistent with the code soon, I just can't figure out how to use the `@test_approx_equal_eps` with a tuple return instead of the array...